pkg/chunk/chunk_store.go

```diff
@@ -228,7 +228,49 @@ func (c *store) LabelValuesForMetricName(ctx context.Context, from, through mode
 		}
 		result = append(result, string(labelValue))
 	}
+	sort.Strings(result)
```

Comment:
why did this need to change?

Reply:
the diff makes you think I did but in fact it's just the same 3 last line for both method, so nothing changed here.